org-code-javabuilder/theater/src/test/java/org/code/theater/SceneTest.java

```diff
@@ -0,0 +1,196 @@
+package org.code.theater;
+
+import static org.code.theater.Scene.*;
+import static org.code.theater.support.DrawImageAction.UNSPECIFIED;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+
+import org.code.media.Color;
+import org.code.media.Font;
+import org.code.media.FontStyle;
+import org.code.media.Image;
+import org.code.theater.support.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class SceneTest {
+
+  private Scene unitUnderTest;
+
+  @BeforeEach
+  public void setUp() {
+    unitUnderTest = new Scene();
+  }
+
+  @Test
+  public void testAllActions() {
```

Comment:
This unit test would fail if any action was updated. Break this up into individual units so it's clearer which piece is failing when a code change happens.